src/script/client/ClientRepository.js

```diff
@@ -336,15 +337,15 @@ z.client.ClientRepository = class ClientRepository {
     let deviceModel = platform.name;
 
     if (z.util.Environment.desktop) {
-      let identifier;
+      let modelString;
       if (z.util.Environment.os.mac) {
-        identifier = z.string.wireMacos;
+        modelString = t('wireMacos');
       } else if (z.util.Environment.os.win) {
-        identifier = z.string.wireWindows;
+        modelString = t('wireWindows');
       } else {
-        identifier = z.string.wireLinux;
+        modelString = t('wireLinux');
       }
-      deviceModel = z.l10n.text(identifier);
+      deviceModel = modelString;
```

Comment:
Please re factor this and call the `t` function only in a single place (like it was before)